src/main/logging/README.md

```diff
@@ -0,0 +1,244 @@
+## Deploy Logging Pattern - EFK Stack
```

Comment:
I would put on the top a disclaimer that this is not officially supported document and it is meant as a guidance.

```
## Disclaimer

**This is not officially supported functionality.** This document serves as an example for users that would like to enable aggregated logging in their Kubernetes cluster. There are many solutions how to provide this capability and this document just showcases some of the options.
```
or something in similar fashion.